src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -81,6 +85,7 @@ internal partial class LanguageServiceHost : OnceInitializedOnceDisposedAsync, I
         [AppliesTo(ProjectCapability.CSharpOrVisualBasicOrFSharpLanguageService)]
         private Task OnProjectFactoryCompletedAsync()
         {
+            _designBuildTelemetryService.OnDesignTimeBuildQueued();
```

Comment:
This is only called once per project. The other methods are called multiple times during a project's lifetime.